src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1283,6 +1283,10 @@ private ArrayTypeSymbol VisitArrayInitializer(BoundArrayCreation node)
                 {
                     bestType = BestTypeInferrer.InferBestType(resultTypes, _conversions, useSiteDiagnostics: ref useSiteDiagnostics);
                 }
+                else
+                {
+                    bestType = TypeSymbolWithAnnotations.Create(bestType.TypeSymbol, isNullableIfReferenceType: null);
```

Comment:
> bestType = TypeSymbolWithAnnotations.Create(bestType.TypeSymbol, isNullableIfReferenceType: null); [](start = 19, length = 99)

I didn't understand this change. I'll stop by before standup.

Reply:
Added comment.

---
In reply to: [210361033](https://github.com/dotnet/roslyn/pull/29317#discussion_r210361033) [](ancestors = 210361033)